src/main/java/uk/ac/ebi/embl/api/validation/submission/SubmissionValidator.java

```diff
@@ -39,9 +36,36 @@ public SubmissionValidator(SubmissionOptions options) {
     }
 
     public void validate() throws ValidationEngineException {
-        new SubmissionValidationPlan(options).execute();
+
+        ValidationPlanResult planResult;
+        try {
+            planResult = new SubmissionValidationPlan(options).execute();
+        } catch (ValidationEngineException e) {
+            writeErrorText(e.getMessage());
+            throw e;
+        }
+
+       if(planResult.hasError()) {
+           if(options.isWebinCLI) {
+               writeErrorText(planResult.getValidationMessage().getOriginText());
+              throw new ValidationEngineException(planResult.getValidationMessage().getOriginText(), ValidationEngineException.ReportErrorType.VALIDATION_ERROR);
+           } else {
+               StringBuilder sb = new StringBuilder();
+               for (ValidationMessage<Origin> error : planResult.getMessages(Severity.ERROR)) {
+                   sb.append(error.getMessage());
+                   sb.append("\n");
+               }
+               throw new ValidationEngineException(StringUtils.chomp(sb.toString()), ValidationEngineException.ReportErrorType.VALIDATION_ERROR);
+           }
+       }
     }
```

Comment:
I guess this is the point we feel we must throw an exception to Webin-CLI. We could throw an exception that wraps the validationResult e.g. ValidationResultException that just wraps the ValidationResult (we could put this next to ValidationException).